packages/runtime/plugin-garfish/src/cli/template.ts

```diff
@@ -0,0 +1,57 @@
+const genRenderCode = ({
+  srcDirectory,
+  internalSrcAlias,
+  metaName,
+  entry,
+  customEntry,
+  mountId,
+}: {
+  srcDirectory: string;
+  internalSrcAlias: string;
+  metaName: string;
+  entry: string;
+  customEntry?: string | false;
+  mountId?: string;
+}) =>
+  customEntry
+    ? `import '${entry.replace(srcDirectory, internalSrcAlias)}'
+export * from '${entry.replace(srcDirectory, internalSrcAlias)}'`
```

Comment:
if we already export `provider` in line 29, do we also need to `export *` here ?

Reply:
when use customEntry, exports provider need user to export?